ember-intl/addon/services/intl.d.ts

```diff
@@ -21,7 +21,7 @@ export interface TOptions {
 type FormatterProxy<T extends keyof IntlService['_formatters']> = (
   value: Parameters<IntlService['_formatters'][T]['format']>[1],
   formatOptions?: Parameters<IntlService['_formatters'][T]['format']>[2] & {
-    locale?: string | [string, ...string[]];
+    locale?: string | string[];
```

Comment:
After merging the PR, I realized that `[string, ...string[]]` might have been used to indicate that the array has at least 1 element.

At a later point, consider reverting the type after refactoring the code in the addon more.